### ISSUE TITLE
Flux forced - varying wind stress by a certain factor

### ISSUE THREAD

**Issue by dhruvbhagtani** [dhruvbhagtani/sfc-perturbation-expts#38]
We have successfully created a set of simulations where changing one doesn't affect the other. In the following issue, we discuss a set of experiments where we change the wind stress, keeping the heat flux the same, as shown in the animation. We run the control, 0.5xS (0.5 times the wind stress compared to control) and 1.5xS (1.5 times the wind stress compared to control) for 100 years each.

https://user-images.githubusercontent.com/29700296/153521773-65ad6fbe-f049-46ef-bda6-8610cf183d82.mp4

**Comment by dhruvbhagtani** [comment]
All results can be found over here: https://github.com/dhruvbhagtani/varying-surface-forcing/blob/main/025deg_flux_expts/Flux_vary_stress.ipynb

**Comment by dhruvbhagtani** [comment]
We observe a systematic increase in gyre strength as we increase the wind stress curl. This is consistent with the Sverdrup theory:

 
![Github_upload 001](https://user-images.githubusercontent.com/29700296/153539842-698abd18-94b1-48eb-b066-94add66077e6.jpeg)

**Comment by navidcy** [comment]
> We observe a systematic increase in gyre strength as we increase the wind stress curl. This is consistent with the Sverdrup theory:
> 
> ![Github_upload 001](https://user-images.githubusercontent.com/29700296/153521738-2f22f78b-2379-4362-b867-88423ee25c05.jpeg)

Sure. But not proportional as it seems. I mean, 53 ≠ 44 x 1.5 and 32 ≠ 44 x 0.5

Suggestion: 32Sv, 44Sv, 53Sv would be great for the figure purposes. You don't need much more accuracy. Anyway how you measure these transports is a bit arbitrary.

**Comment by navidcy** [comment]
Figures look great!

Another minor point of order from the ocd figure-critic is that the colorbar should have as many levels as the number of levels in the figures. Then it's obvious what value corresponds to which contour.

**Comment by navidcy** [comment]
What are these red contours?

**Comment by dhruvbhagtani** [comment]
Good point. The red contours are same for all the cases - it shows the extent of the North Atlantic subtropical gyre in the control run. We superimpose this contour on all cases to understand the spatial differences between gyres of different experiments. 

Also, I have updated my previous comment to make the colorbar consistent with my plot.
